projects/plugins/jetpack/extensions/blocks/contact-info/edit.js

```diff
@@ -25,7 +26,12 @@ const ALLOWED_BLOCKS = [
 	'core/video',
 ];
 
-const TEMPLATE = [ [ 'jetpack/email' ], [ 'jetpack/phone' ], [ 'jetpack/address' ] ];
+const TEMPLATE = [
+	[ 'jetpack/name' ],
```

Comment:
It seems better to create a new children block (rather than using, for instance, `core/paragraph`) to have a finer control on the markup rendered in the frontend.